### ISSUE TITLE
Decide Whether To Do An Implementation Or A Paper  

### ISSUE THREAD

**Issue by andydmh** [andydmh/Machine-Learning-Group9#22]
Ideas or suggestions about what to do for this project. Post your comments bellow.

**Comment by jbowren** [comment]
For an implementation, how about digit recognition with SKLearn's load_digits? I've trained a MLP to recognize handwritten letters of "A," and "B," but had problems with other letters. Maybe digits would work fine. We could use sklearn.neural_network.MLPClassifier for a comparison.

For a more recent paper, we could analyze Krizkevsky's convolutional neural network [paper.](http://papers.nips.cc/paper/4824-imagenet-classification-with-deep-convolutional-neural-networks.pdf) Slightly easier (and older) is Hinton's [paper](http://science.sciencemag.org/content/313/5786/504.full?casa_token=1YTObJr3u5IAAAAA:IV66LuNom_lcgWFM7U_XmY5g3NyRRBdRxZT96wOEvLrHejN7y-y4a2dlnJ-BieUnTiP2Bz452H0tN9w) on autoencoders. Easiest of all would be something about multilayer perceptrons in general.

**Comment by andydmh** [comment]
I added a [wiki link](https://github.com/andydmh/Machine-Learning-Group9/wiki/Deep-Learning) with some famous papers on Deep Learning for the purpose of picking an interesting one. Notice that Deep Learning uses the concepts covered in class about Neural Networks but it is an entire discipline on its own. Lets try to pick something simple and interesting. I also included on the list the paper mentioned by @jbowren since it is a well known architecture. Post your feedbacks so that we can pick one and start working as soon as posible (also if you have another interesting paper add them to the list).
